src/Shared/FileMatcher.cs

```diff
@@ -1905,21 +2123,42 @@ DirectoryExists directoryExists
              * This is because it's cheaper to add items to an IList and this code
```

Comment:
Comment is outdated.

Reply:
[Updated](https://github.com/maca88/msbuild/blob/b5574289426c99e9567a43fe17d4731e1f4f5380/src/Shared/FileMatcher.cs#L2188)